sdk/sdk.go

```diff
@@ -198,7 +198,7 @@ func (babylonClient *babylonQueryClient) queryFpPower(fpPubkeyHex string, btcHei
 	return totalPower, nil
 }
 
-func (babylonClient *babylonQueryClient) QueryIsBlockBabylonFinalized(queryParams QueryParams) (bool, error) {
+func (babylonClient *BabylonQueryClient) QueryIsBlockBabylonFinalized(queryParams QueryParams) (bool, error) {
```

Comment:
```suggestion
func (babylonClient *BabylonQueryClient) QueryIsBlockBabylonFinalized(queryParams *QueryParams) (bool, error) {
```

Also `QueryParams` basically contains necessary metadata of a L2 block, so its naming could be sth like `L2Block`